src/node_options.h

```diff
@@ -115,6 +115,11 @@ class EnvironmentOptions : public Options {
   uint64_t cpu_prof_interval = kDefaultCpuProfInterval;
   std::string cpu_prof_name;
   bool cpu_prof = false;
+  std::string heap_prof_dir;
+  std::string heap_prof_name;
+  static const uint64_t kDefaultHeapProfInterval = 32 * 1024;
```

Comment:
Is there a reason for using this particular value here? IMO, we should use the default in V8 unless we have a good reason to say that most Node.js workloads would benefit from a significantly lower interval.

FYI, the default in V8 is 512KiB.

Reply:
@ofrobots The default in the inspector seems different, from https://chromedevtools.github.io/devtools-protocol/v8/HeapProfiler#method-startSampling

> Average sample interval in bytes. Poisson distribution is used for the intervals. The default value is 32768 bytes.

Reply:
For Node.js, I think the V8 threshold makes more sense. I added it with longer running production workloads in mind. Frequent intervals have more overhead, but 32k is large enough that maybe the difference is not perceptible.

Reply:
Thanks for the explanation, I think we can just use 512KB and if anyones files an issue about this being too high, we can just revisit.